workspaces/diff-engine/src/projections/conflicts.rs

```diff
@@ -0,0 +1,345 @@
+use crate::events::{EndpointEvent, Event, ShapeEvent, SpecEvent};
```

Comment:
🍖

Reply:
Looks good, exactly how and where I'd expect a projection like this to exist. The approach tracking node id's and domain id's hard to fault as well, in line with how the other projections work, too!